CHANGELOG.md

```diff
@@ -3,6 +3,25 @@
 
 # Rivet Changelog
 
+## 0.23.0
+
+### Changed
+
+* `Export-Migration` no longer creates an `Add-Schema` operation that includes owner information (i.e. the `Owner`
+parameter is omitted). When exporting a schema to DDL in SQL Server Management Studio, the owner information is also

```

Comment:
I would write this the other way around because by putting "_no longer creates an `Add-Schema` operation_" first, it sounds like that was what was removed. I had to read it a couple of times to understand.

E.g. something more like:

> `Export-Migration` no longer includes owner information when it creates an `Add-Schema` operation (i.e. the `Owner`
parameter is omitted).